updated steps used for installing pecan using docker (#2617)
PecanProject/pecan: book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

    b. "Initialize" the data for the PEcAn database.

        ```bash
        docker-compose run --rm bety initialize
        docker run --rm --network pecan_pecan pecan/db
        ```
        
        This should produce a lot of output describing the database operations happening under the hood.

>         Some of these will look like errors (including starting with `ERROR`), but _this is normal_.

Reviewer: Is this line removed because no error-like messages are expected anymore, or for some other reason? This line has reassured me on several occasions and I'd vote to keep it unless the expected output is now truly warning-free.

Reviewer 2: there are still error messages but not as obvious as before and they don't start with `ERROR` anymore, maybe we can just delete the parenthesis bit (because it could also be confusing to not see messages starting with `ERROR` at first glance)
```

docker run --rm --network pecan_pecan pecan/db

Unable to find image 'pecan/db:latest' locally
latest: Pulling from pecan/db
cbdbe7a5bc2a: Pull complete 
0826c3db658a: Pull complete 
1e6696f14f8a: Pull complete 
Digest: sha256:5cb1c84ac2afaa3e7e4f78fbd6e55d5d24822cea6b0e29294df3014173c41212
Status: Downloaded newer image for pecan/db:latest
CREATE ROLE
pg_restore: connecting to database for restore
pg_restore: while INITIALIZING:
pg_restore: error: could not execute query: ERROR:  unrecognized configuration parameter "idle_in_transaction_session_timeout"
Command was: SET idle_in_transaction_session_timeout = 0;
pg_restore: dropping DATABASE bety
pg_restore: while PROCESSING TOC:
pg_restore: from TOC entry 4783; 1262 20600 DATABASE bety bety
pg_restore: error: could not execute query: **ERROR**:  database "bety" does not exist
Command was: DROP DATABASE bety;
pg_restore: creating DATABASE "bety"
pg_restore: connecting to new database "bety"
pg_restore: connecting to database "bety" as user "postgres"
pg_restore: error: could not execute query: **ERROR**:  unrecognized configuration parameter "idle_in_transaction_session_timeout"
Command was: SET idle_in_transaction_session_timeout = 0;
pg_restore: creating SCHEMA "admin"
pg_restore: creating SCHEMA "public"
pg_restore: from TOC entry 8; 2615 22127 SCHEMA public bety
pg_restore: error: could not execute query: **ERROR**:  schema "public" already exists
Command was: CREATE SCHEMA public;
```

Author: Ok....so I am adding a line that says: "Some of these will look like errors, but _this is normal_.". I think this will address the fact that some statements actually do have the word "error" in them, but also that they do not start with the word "error"

>         This command succeeded if the output ends with the following (the syntax for creating a new user for accessing BetyDB: 

Reviewer 2: ```suggestion
        This command succeeded if the output ends with the following (the syntax for creating a new user for accessing BetyDB): 
```

    b. "Initialize" the data for the PEcAn database.

        ```bash
        docker run --rm --network pecan_pecan pecan/db
        ```
        
        This should produce a lot of output describing the database operations happening under the hood.

> <<<<<<< HEAD

Reviewer 2: do you have a conflict?

Author: I thought I resolved this locally...didn't realize how it ended up here. Will fix it

        Done!
        ###################################################################### 
        ```

>     e. Download the [`pecan/docker/env.example`](https://raw.githubusercontent.com/PecanProject/pecan/develop/docker-compose.yml) & save it as `.env` file.

Reviewer 2: ```suggestion
    e. Download the [`pecan/docker/env.example`](https://raw.githubusercontent.com/PecanProject/pecan/develop/docker/env.example) & save it as `.env` file.
```

>        Now, open the `.env` file & uncomment the lines mentioned below:

Reviewer 2: I liked the idea of showing this visually, but I think documentation should explain why we need to uncomment traefik configuration flags (I personally don't uncomment them)

Author: Ok...will speak with @robkooper and write about the explanation for uncommenting these lines

Reviewer 3: I agree, we can do that in a later section about configuring traefik. This should be just about getting pecan up and running.

Author: So...I suppose we can leave this part as is? And include details about traefik later in the documentation @istfer ?

Reviewer 2: by "as is" do you mean leaving the PR as is, with the screenshot that `TRAEFIK` flags uncommented? I thought @robkooper also agreed that this bit shouldn't involve anything related to traefik configuration

Author: I thought @robkooper mentioned that we show the user which lines to uncomment (using this image), and in a later part of the documentation , explain about traefik configurations.

Coz if we don't show the user what to uncomment at this stage of the installation, I'm afraid (s)he will have trouble in this stage of installation.

Reviewer 2: I agree that it is nice to show which lines to uncomment, but I never uncomment TRAEFIK tags for my own set up (which might be something I'm doing wrong). That's why I was curious if this is something everyone must do, if yes it needs explanation, if not the screenshot is a bit misleading at this stage.

Overall, I'm with it staying as well.

Reviewer 3: I think it would be best to only highlight the minimal work that needs to be done, in this case 2 variables
```
COMPOSE_PROJECT_NAME=pecan
PECAN_VERSION=develop
```
I think we should clarify that only set `PECAN_VERSION` if you want to run the develop branch, meaning it can have bugs.

Author: I have made required modifications...pls have a look @istfer @robkooper

       ```{r, echo=FALSE, fig.align='center'}
       knitr::include_graphics(rep("figures/env-file.PNG"))
       ```
       

>        Setting `PECAN_VERSION=develop` indicates that you want to run the bleeding-edge `develop` branch, meaning it may have bugs. To go ahead with the stable version you may set `PECAN_VERSION=latest` or `PECAN_VERSION=<release-number>` (For example `1.7.0`). [Here](https://github.com/pecanproject/pecan/releases) are the details about all the PEcAn releases.

Reviewer 3: ```suggestion
       Setting `PECAN_VERSION=develop` indicates that you want to run the bleeding-edge `develop` branch, meaning it may have bugs. To go ahead with the stable version you may set `PECAN_VERSION=latest` (master branch on github) or `PECAN_VERSION=<release-number>` (For example `1.7.0`).  You can look at the list of all  [releases](https://github.com/pecanproject/pecan/releases) of PEcAn to see what options are availble.
```
